web/src/features/fireWeather/components/graphs/WxDataGraph.tsx

```diff
@@ -1,14 +1,16 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { makeStyles } from '@material-ui/core/styles'
+import moment from 'moment'
 
 import { ModelSummary, ModelValue } from 'api/modelAPI'
 import { ObservedValue } from 'api/observationAPI'
 import { NoonForecastValue, ForecastSummary } from 'api/forecastAPI'
 import WxDataGraphToggles from 'features/fireWeather/components/graphs/WxDataGraphToggles'
 import { useGraphToggles } from 'features/fireWeather/components/graphs/useGraphToggles'
-import TempRHGraph from 'features/fireWeather/components/graphs/TempRHGraph'
-import PrecipGraph from 'features/fireWeather/components/graphs/PrecipGraph'
+import PrecipGraph from 'features/fireWeather/components/graphs/PercipGraph'
```

Comment:
```suggestion
import PrecipitationGraph from 'features/fireWeather/components/graphs/PrecipitationGraph'
```

I don't feel too strongly about not abbreviating, but definitely fix spelling.

Reply:
PrecipitationGraph, Done